### ISSUE TITLE
Log out client version often

### ISSUE THREAD

**Issue by aranhoide** [getlantern/lantern#3508]
Some log files we get sent don't capture client startup, so it's hard to tell what client version the files correspond to.  A goroutine that just logs the client version (say) every minute would help here.

Please close this if we already have any other way to get this info.

**Comment by myleshorton** [comment]
Yeah I think it's because they roll over across log files, so we sometimes don't get it, but agreed it would be useful! In theory everyone's getting auto-updated, but you never know.
